tls/tests/test_remote.py

```diff
@@ -335,3 +347,95 @@ def test_mysql_ok(aggregator, instance_remote_mysql_valid):
     aggregator.assert_metric('tls.days_left', count=1)
     aggregator.assert_metric('tls.seconds_left', count=1)
     aggregator.assert_all_metrics_covered()
+
+
+def test_valid_version_with_critical_certificate_validation_and_critial_certificate_expiration(
+    aggregator, instance_remote_ok
+):
+    if PY2:
+        # Skip this test on Python 2, no longer in active support
+        return
```

Comment:
You should use https://github.com/DataDog/integrations-core/blob/970de2dc9946690ab703ce87d02ffb959ed4da31/datadog_checks_dev/datadog_checks/dev/testing.py#L16

Reply:
Thanks, this is nice decorator 👍